electron/index.js

```diff
@@ -100,11 +100,17 @@ class CapacitorSplashScreen {
       frame: false,
       show: false,
       transparent: this.splashOptions.transparentWindow,
+      webPreferences: {
+        // Required to load file:// splash screen
+        webSecurity: false  
+      }
```

Comment:
I figure this is safe as long as we're only loading a splash screen. It might make more sense to base this value on whether or not the developer provided customHtml, which cannot be verified.